ecosystem/sep-0012.md

```diff
@@ -315,3 +316,38 @@ Situation | Response
 Success | `200 OK`
 Client not authenticated properly | `401 Unauthorized`
 Anchor has no information on the customer | `404 Not Found`
+
+## Customer callback PUT
```

Comment:
Can we remove the newline characters in this section to match the style of the rest of the doc?